src/components/AddTask/AddTask.jsx

```diff
@@ -1,17 +1,17 @@
 import React from "react";
-import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
 
 import { AddBtn, VerticalLine, HorizontalLine } from "./AddTask.style";
 
-const AddTask= ({addDayToTask}) => {
+const AddTask = ({ addDayToTask }) => {
     return <AddBtn onClick={addDayToTask}>
         <VerticalLine></VerticalLine>
         <HorizontalLine></HorizontalLine>
     </AddBtn>
 }
 
 AddTask.propTypes = {
-    addDayToTask: PropTypes.func.isRequired
+    // addDayToTask: PropTypes.func.isRequired
```

Comment:
uncommit it pls

Reply:
Yes, of course) I will remove the comment in the next Pool Request